x/txfees/types/keys.go

```diff
@@ -10,11 +14,28 @@ const (
 	// RouterKey is the message route for slashing
 	RouterKey = ModuleName
 
+	// FeeCollectorName the module account name for the fee collector account address.
+	FeeCollectorName = "fee_collector"
+
+	// AltFeeCollectorName the module account name for the alt fee collector account address (used for auto-swapping non-OSMO tx fees).
+	AltFeeCollectorName = "alt_fee_collector"
```

Comment:
Should we rename this to "NonNativeFeeCollector" ?

Reply:
Agree with the renaming!